spec/models/user_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:email) { 'robert@example.com' }
+  let(:first_name) { 'robert' }
+  let(:last_name) { 'rodriguez' }
+  let(:user) { create_user(email, first_name, last_name) }
```

Comment:
Couldn't this just be a `User.new` like the other `:user` var declarations calls `User.create!`? If you need it to not lazy-load you can use `let!(:user)` and it will declare the var at the beginning of the file running.